datafusion/physical-expr/Cargo.toml

```diff
@@ -44,6 +44,7 @@ arrow-schema = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
 datafusion-expr-common = { workspace = true }
+datafusion-functions = { workspace = true }
```

Comment:
since only test requires this, maybe move to dev-dependencies

Reply:
or maybe move the test in `core/test` 🤔

Reply:
I've moved it to dev-dependencies for now